src/constructUrl.js

```diff
@@ -5,9 +5,9 @@ Licensed under the Apache License 2.0, seen https://github.com/coursera/react-im
 Minor syntax modifications have been made
 */
 
-var Base64 = require("js-base64").Base64;
 const PACKAGE_VERSION = require("../package.json").version;
```

Comment:
Let's change this to using `common/constants.js`, must have slipped through the cracks.